Description: Correct pod NAME issues in LedgerSMB/AA.pm
 Has the module name but does not have the dash plus short description in the
 NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472504&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/AA.pm
+++ b/LedgerSMB/AA.pm
@@ -2,7 +2,8 @@
 
 =head1 NAME
 
-LedgerSMB::AA
+LedgerSMB::AA - Contains the routines for managing AR and AP transaactions,
+              and also many of the reports.
 
 =head1 SYNPOSIS
 
